CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_hubspot v0.20.0
+[PR #150](https://github.com/fivetran/dbt_hubspot/pull/150) includes the following updates:
+
+## Breaking Change (`--full-refresh` required after upgrading)  
+- Introduced the new `category` column to certain upstream models and added to relevant unique tests. This association field differentiates records by either HUBSPOT_DEFINED (default label) or USER_DEFINED (custom label) and was introduced to the connector in October 2024. For more information, refer to the upstream [CHANGELOG](https://github.com/fivetran/dbt_hubspot_source/releases/tag/v0.17.0)
```

Comment:
I know it's exhaustive, but can you include essentially the entire breaking change note from the source CHANGELOG to be here as well. We can still link to the source release notes, but since this is what is rendered in the Quickstart UI, it would be great to show all relevant breaking changes here without requiring the customer to click through a new link.

Reply:
no prob, that makes sense! added the note from source in its entirety